### PR TITLE
Fix https://github.com/Microsoft/TypeScript/issues/19270: ensure output name is a valid locale name

### PR DESCRIPTION
**Pull request by mhegazy**
Follow up on https://github.com/Microsoft/TypeScript/pull/18702.

In the original iteration, I have wrongly produced the output in a folder with the same name as the input. the input folder names are [3-letter abbreviations](https://msdn.microsoft.com/en-us/library/aa246450(v=vs.60).aspx) used by VS, but the output should be valid [locale names](https://msdn.microsoft.com/en-us/library/windows/desktop/dd373814(v=vs.85).aspx).

Fixes https://github.com/Microsoft/TypeScript/issues/19270

### CODE REVIEW

File: Jakefile.js [hunk 1]
```diff
@@ -239,6 +239,10 @@ var libraryTargets = librarySourceMap.map(function (f) {
     return path.join(builtLocalDirectory, f.target);
 });
 
+var localizationTargets = ["cs", "de", "enu", "es", "fr", "it", "ja", "ko", "pl", "pt-BR", "ru", "tr", "zh-CN", "zh-TW"].map(function (f) {
```
Comment (paulvanbrenk) [inline Jakefile.js]: `enu` should be `en`

Comment (mhegazy) [reply Jakefile.js]: that is where the lcg file lives. we do not have an lcl file for it.

Comment (mhegazy) [reply Jakefile.js]: Added a comment explaining the process

**Comment by paulvanbrenk** [comment]
👍

**Pull request merged**